deeppavlov/models/multitask_pal_bert/multitask_pal_bert.py

```diff
@@ -0,0 +1,468 @@
+                raise ConfigError("Provided load path is incorrect!")
+
+            weights_path = Path(self.load_path.resolve())
+            weights_path = weights_path.with_suffix(f".pth.tar")
+            if weights_path.exists():
+                log.info(f"Load path {weights_path} exists.")
+                log.info(
+                    f"Initializing `{self.__class__.__name__}` from saved.")
+
+                # firstly, initialize with random weights and previously saved
+                # parameters
+                self.init_from_opt()
+
+                # now load the weights, optimizer from saved
+                log.info(f"Loading weights from {weights_path}.")
+                checkpoint = torch.load(weights_path, map_location=self.device)
+                self.model.load_state_dict(checkpoint["model_state_dict"])
+                self.optimizer.load_state_dict(
+                    checkpoint["optimizer_state_dict"])
+                self.epochs_done = checkpoint.get("epochs_done", 0)
+            else:
+                log.info(
+                    f"Init from scratch. Load path {weights_path} does not exist.")
+                self.init_from_opt()
+        else:
+            log.info(
+                f"Init from scratch. Load path {self.load_path} is not provided.")
+            self.init_from_opt()
+
+        if self.freeze_embeddings:
+            for n, p in self.model.bert.named_parameters():
+                if (
+                    "aug" in n
+                    or "classifier" in n
+                    or "mult" in n
+                    or "gamma" in n
+                    or "beta" in n
+                ):
+                    continue
+                p.requires_grad = False
+                log.info("Bert Embeddings Freezed")
+
+    def __call__(self, *args):
+        """Make prediction for given features (texts).
+        Args:
+            features: batch of InputFeatures for all tasks
+        Returns:
+            predicted classes or probabilities of each class
+        """
+        self.validation_predictions = []
+        n_in = sum([inp for inp in self.in_distribution.values()])
+
+        features = args[1:]
+        args_in = features[:n_in]
+        in_by_tasks = self._distribute_arguments_by_tasks(
+            args_in, {}, self.task_names, "in"
+        )
+
+        for task_id in range(len(self.task_names)):
+            task_features = in_by_tasks[self.task_names[task_id]]
+
+            _input = {}
+            for elem in ["input_ids", "attention_mask", "token_type_ids"]:
+                _input[elem] = [getattr(f, elem) for f in task_features[0]]
+                _input[elem] = torch.cat(_input[elem], dim=0).to(self.device)
+
+            with torch.no_grad():
+                tokenized = {
+                    key: value
+                    for (key, value) in _input.items()
+                    if key in self.model.forward.__code__.co_varnames
```

Comment:
https://github.com/deepmipt/DeepPavlov/pull/1463/files#r686792642

Reply:
Removed since it is not required.